src/coreclr/vm/appdomain.cpp

```diff
@@ -2923,23 +2915,23 @@ struct LoadFileArgs
 {
     FileLoadLock *pLock;
     FileLoadLevel targetLevel;
-    DomainFile *result;
+    DomainAssembly *result;
```

Comment:
Is anything actually using `LoadFileArgs`?

Reply:
Nope. It is dead code.